### ISSUE TITLE
SA1008, SA1012 false positives for property pattern inside tuple deconstruction

### ISSUE THREAD

**Issue by jnm2** [DotNetAnalyzers/StyleCopAnalyzers#3141]
Using 1.2.0-beta.164

```cs
class C
{
    void M((string A, string B) tuple)
    {
        // ⚠ SA1008 Opening parenthesis should not be preceded by a space.
        // ⚠ SA1012 Opening brace should be preceded by a space
        //           ↓↓
        _ = tuple is ({ Length: 1 }, { Length: 2 });
    }
}
```

**Comment by jnm2** [comment]
Hey, is there a prize for when your issue number lands on the beginning digits of π?
